src/main/webapp/scripts/nearbyVendors.js

```diff
@@ -103,6 +125,37 @@ const addVendorsToMap = (map, nearbyVendors) => {
   });
 };
 
+/**
+ * Highlights marker of the vendor in the map
+ * @param {Element} Card element container of the vendor
+ */
+function showMarker(cardContainer) {
```

Comment:
```suggestion
function highlightMarker(cardContainer) {
```
Nit: Maybe change the name of this to match the comment description.